src/components/NewFooter/MenuContent.tsx

```diff
@@ -46,7 +46,7 @@ const MenuContent: React.FC = () => {
         ))}
       </Section>
       <Section>
-        <LogoWrapper to="/">
+        <LogoWrapper to="/" aria-label="Covid Act Now">
           <LogoNonUrl />
```

Comment:
FWIW- An alternative fix might be to add a `title` to the LogoNonUrl `<svg>` tag, which I think is the svg-equivalent of `alt` text on an image.  But the aria-label on the link seems fine too!

Reply:
`title` is not recommended in most cases because it's inconsistently handled by screen readers

Reply:
Ah, excellent to know. Thanks!